### PR TITLE
Collector token

### PR DESCRIPTION
**Pull request by shriramshankar**
generate a new "collector" based token and send it as a response to post collector endpoint.

### CODE REVIEW

**Comment by shriramshankar** [comment]
I have 2 more PRs coming up as a part of this story.

File: api/v1/controllers/collectors.js [hunk 1]
```diff
@@ -37,8 +35,23 @@ const ZERO = 0;
  * @param {Function} next - The next middleware function in the stack
  */
 function postCollector(req, res, next) {
-  // TODO reject if caller is not a collector
-  doPost(req, res, next, helper);
+  const collectorToPost = req.swagger.params.queryBody.value;
```
Comment (iamigo) [inline api/v1/controllers/collectors.js]: Pls. add a comment describing the extra work being done here, e.g. "When a collector registers itself with Refocus, Refocus sends back a special token for that collector to use for all further communication."

Comment (shriramshankar) [inline api/v1/controllers/collectors.js]: @iamigo Can you approve this,  I will fix this in the next PR?

**Comment by shriramshankar** [comment]
merging this PR

**Pull request merged**